libraries/AP_HAL_ChibiOS/UARTDriver.cpp

```diff
@@ -430,6 +430,18 @@ void UARTDriver::_begin(uint32_t b, uint16_t rxS, uint16_t txS)
             sercfg.cr2 = _cr2_options;
             sercfg.cr3 = _cr3_options;
 
+#if defined(STM32H7)
+            /*
+              H7 defaults to 16x oversampling. To get the highest
+              possible baudrates we need to drop back to 8x
+              oversampling. The H7 UART clock is 100MHz. This allows
+              for up to 12.5MBps on H7 UARTs
```

Comment:
Guessing you probably mean 12.5Mbps, as in 12.5 million bits per second? Capital B is usually bytes.

Reply:
yep, sorry.